File: YukariLauncher/src/main/java/net/kdt/pojavlaunch/progresskeeper/ProgressState.java
package net.kdt.pojavlaunch.progresskeeper;

/**
 * 進行状況の状態を保持するデータクラス。
 */
public class ProgressState {
    /** 現在の進行度 */
    int progress;
    /** リソースID */
    int resid;
    /** 可変長引数 */
    Object[] varArg;
}

File: YukariLauncher/src/main/java/com/arata/yukarilauncher/task/ProgressState.kt
package com.arata.yukarilauncher.task

/** 進捗状態を保持するデータクラス。 */
class ProgressState {
    /** 進捗値 */
    var progress: Int = 0
    /** リソースID */
    var resid: Int = 0
    /** 追加引数 */
    var varArg: Array<out Any?>? = null
}
